src/main/java/reposense/model/RepoConfiguration.java

```diff
@@ -24,8 +22,6 @@
     private RepoLocation location;
     private String branch;
     private String displayName;
-    private Date sinceDate;
-    private Date untilDate;
```

Comment:
Why are we removing this 2 variables from `RepoConfiguration`? 
These are part of the configuration settings, and should still remain here.
Removing this 2 ends up causing lots of redundant changes throughout the code, as we have to manually pass this 2 values down alongside the `RepoConfiguration`, which is unnecessary.

If we don't need these 2 to be printed out in the 'json' file, we can just set them both as `transient` instead.

Reply:
@yamidark fixed